### DISCUSSION TITLE
Where is my "autolink references" option?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#52699] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Why can't I see the "Autolink References" option under my repo Settings > Integration?

This repo is under a organization that I am the owner of and we would like to have autolink refs to Jira.

What am I missing? 

![autolinkref](https://user-images.githubusercontent.com/129647066/231689335-684f8d31-262f-4c03-b857-df97f4d72099.png)

**Comment by JonathanCabezas** [comment]
I'm bumping this issue, [the documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources#configuring-autolinks-to-reference-external-resources) **clearly states** this should be **here**.

Edit: It looks like it's not a feature in **Private Repositories** unless you have a [Pro Account](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-pro).
This made me update my account, such a valuable feature for using an external **Task Tracker**, thanks Github.
